server/app/services/export/JsonExporter.java

```diff
@@ -289,15 +293,6 @@ private static boolean instanceOfNonEmptyImmutableListOfString(Object value) {
     return !list.isEmpty() && list.get(0) instanceof String;
   }
 
-  // Returns true if value is an empty ImmutableList<>.
-  private static boolean instanceOfEmptyImmutableList(Object value) {
```

Comment:
This was used for unanswered mutli-select questions, which returned an empty array. That's not handled on line 263 of this file, but checking for an array element path with an empty optional.